src/library/dlgtrackinfo.cpp

```diff
@@ -400,21 +402,22 @@ void DlgTrackInfo::populateCues(TrackPointer pTrack) {
         // Make the type read only
         typeItem->setFlags(Qt::NoItemFlags);
 
+        HotcueColorPaletteSettings colorPaletteSettings(m_pConfig);
+        auto colorPalette = colorPaletteSettings.getHotcueColorPalette();
+        QList<QColor> hotcueColorList =
+                colorPaletteSettings.getHotcueColorPalette().m_colorList;
         QComboBox* colorComboBox = new QComboBox();
-        const QList<PredefinedColorPointer> predefinedColors = Color::kPredefinedColorsSet.allColors;
-        for (int i = 0; i < predefinedColors.count(); i++) {
-            PredefinedColorPointer color = predefinedColors.at(i);
-            QColor defaultRgba = color->m_defaultRgba;
-            colorComboBox->addItem(color->m_sDisplayName, defaultRgba);
-            if (*color != *Color::kPredefinedColorsSet.noColor) {
-                QPixmap pixmap(80, 80);
-                pixmap.fill(defaultRgba);
-                QIcon icon(pixmap);
-                colorComboBox->setItemIcon(i, icon);
-            }
+        for (int i = 0; i < hotcueColorList.count(); i++) {
+            QColor color = hotcueColorList.at(i);
+            colorComboBox->addItem("", color);
```

Comment:
Will the name come back?
This would be a regression if not.

Reply:
Do we need the name? It will only be displayed on the drop-down menu to select a color.

Hotcues already have a label, so the ability to customize the color name on the palette colors don't seem specially useful to me.

Why do you miss the color names?

Reply:
Since we plan to make it possible to choose arbitrary colors, maintaining a list of names is not really feasible. But the "name" should be something like "#FF0000" IMHO.

Reply:
IMHO having a name for s color helps to remember it. If you have two shades of red, and you want to pick the red, you can select the name. This also helps to emphasis the Mixxx colours as the vavorite optimized colours.

Reply:
If we add names to the colors, then when you add a new color to the palette you will be able to give a name to the color. So, not only the default mixxx palette will have colors with names.

I still don't see this as something we need. The name of the color will help differentiating similar colors on the color picker menu. On the other hand, on the waveform, you won't see the color name, so you shouldn't pick similar colors in the first place. Am I missing some other use case?

Reply:
I don't think we really need names for colors. Let's not complicate the code with features that aren't very helpful.

Reply:
What we need is a hint, that emphasize the default colors. Naming them is a IMHO a good way to solve this.